src/main/java/de/pi3g/pi/oled/OLEDDisplay.java

```diff
@@ -121,43 +134,75 @@
 
     private final I2CBus bus;
     private final I2CDevice device;
+    private final Rotation rotation;
 
     private final byte[] imageBuffer = new byte[(DISPLAY_WIDTH * DISPLAY_HEIGHT) / 8];
 
+    /**
+     * @deprecated Use {@link OLEDDisplay#OLEDDisplay(int, int, Rotation)} instead
+     */
+    public OLEDDisplay(int busNumber, int displayAddress) throws IOException, UnsupportedBusNumberException {
+        this(busNumber, displayAddress, Rotation.DEG_0);
+    }
+
+    /**
+     * @deprecated Use {@link OLEDDisplay#OLEDDisplay(int, Rotation)} instead
+     */
+    public OLEDDisplay(int displayAddress) throws IOException, UnsupportedBusNumberException {
+        this(DEFAULT_I2C_BUS, displayAddress, Rotation.DEG_0);
+    }
+
     /**
      * creates an OLED display object with default
-     * i2c bus 1 and default display address of 0x3C
+     * i2c bus 1, default display address of 0x3C and
+     * and the default orientation
      *
      * @throws IOException
      * @throws com.pi4j.io.i2c.I2CFactory.UnsupportedBusNumberException
      */
     public OLEDDisplay() throws IOException, UnsupportedBusNumberException {
-        this(DEFAULT_I2C_BUS, DEFAULT_DISPLAY_ADDRESS);
+        this(DEFAULT_I2C_BUS, DEFAULT_DISPLAY_ADDRESS, Rotation.DEG_0);
+    }
+
+    /**
+     * creates an OLED display object with default
+     * i2c bus 1, default display address of 0x3C
+     * and a given rotation
+     *
+     * @param rotation orientation of the display, can be used if the display is mounted upright or flipped
+     * @throws IOException
+     * @throws com.pi4j.io.i2c.I2CFactory.UnsupportedBusNumberException
+     */
+    public OLEDDisplay(Rotation rotation) throws IOException, UnsupportedBusNumberException {
+        this(DEFAULT_I2C_BUS, DEFAULT_DISPLAY_ADDRESS, rotation);
     }
 
     /**
      * creates an OLED display object with default
      * i2c bus 1 and the given display address
      *
      * @param displayAddress the i2c bus address of the display
+     * @param rotation orientation of the display, can be used if the display is mounted upright or flipped

```

Comment:
I think this is a typo: one will suffice `orientation` or `rotation`.

Reply:
`rotation` is the parameter name, for which the discription should be:
 `orientation of the display, can be used if the display is mounted upright or flipped`

This will produce the following javadoc: 

![grafik](https://user-images.githubusercontent.com/17405009/79197275-b990e700-7e31-11ea-93bf-133abc17f6f9.png)

If you have a better idea for a description let me know.

Reply:
Ah right - my bad. Of course that's the parameter name ;)